src/components/CascadeMenu/PCascadePanels.vue

```diff
@@ -0,0 +1,102 @@
+<template>
+  <transition :name="transitionName">
+    <div v-show="isOpen" class="p-cascade-panels">
+      <template v-for="{ id } in panels" :key="id">
+        <transition-group :name="panelTransitionName">
+          <PCascadePanel v-show="panelIsOpen(id)" :key="id" :panel-id="id" class="p-cascade-panels__panel">
+            <template #default="panel">
+              <slot :name="`${getBaseSlotName(id)}`" v-bind="panel">
+                {{ getBaseSlotName(id) }}
+              </slot>
+            </template>
+          </PCascadePanel>
+        </transition-group>
+      </template>
+    </div>
+  </transition>
+</template>
+
+<script setup lang="ts">
+  import { inject } from 'vue'
+  import { PCascadePanel } from '@/components'
+  import { CascadePanel, CascadePanelId, cascadePanelsKey, useCascadePanels } from '@/compositions'
+  import { kebabCase } from '@/utilities'
+
+  const props = withDefaults(defineProps<{
+    panels: CascadePanel[],
+    panelTransitionName?: string,
+    transitionName?: string,
+  }>(), {
+    panelTransitionName: 'p-cascade-panels__panel-transition',
+    transitionName: 'p-cascade-panels__transition',
+  })
+
+  // Calling inject with a nullish coallescing operator while providing a default value of
+  // undefined allows callers to use the `useCascadePanels` composition function while
+  // ensuring that the composition state is the same.
+  const {
+    close,
+    closeAll,
+    closePanelById,
+    getPanelById,
+    isOpen,
+    open,
+    openPanelById,
+    openPanels,
+    panelIsOpen,
+    toggle,
+    togglePanelById,
+  } = inject(cascadePanelsKey, undefined) ?? useCascadePanels(() => props.panels)
+
+  defineExpose({
+    close,
+    closeAll,
+    closePanelById,
+    getPanelById,
+    isOpen,
+    open,
+    openPanelById,
+    openPanels,
+    panelIsOpen,
+    toggle,
+    togglePanelById,
+  })
+
+  function getBaseSlotName(panelId: CascadePanelId): string {
+    return `${kebabCase(panelId.toString())}`
+  }
+</script>
+
+<style>
+.p-cascade-panels { @apply
+  flex
+  flex-wrap
+  gap-2
+  pointer-events-none
```

Comment:
Curious why this has `pointer-events-none`

Reply:
There might be another way to handle this but I found in testing that there was often a big difference in the heights/widths of panels which caused the bounding box of this component to be massive. Unless we ignore pointer events on this (you'll see this is reset on the panels themselves), the bounding box invisibly captures click events when using this in an absolutely-positioned context